sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -1031,6 +1031,7 @@ object SQLContext {
    */
   @deprecated("Use SparkSession.setActiveSession instead", "2.0.0")
   def setActive(sqlContext: SQLContext): Unit = {
+    // For code-review, this is the only place that call SparkSession.setActiveSession
```

Comment:
you can put it as a PR comment, not code comment that gets committed to the code base.

Reply:
removed